extensions/wikia/AdEngine/js/video/player/porvata/googleImaPlayerFactory.js

```diff
@@ -8,7 +8,8 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory', [
 	'wikia.window'
 ], function (imaSetup, moatVideoTracker, vastDebugger, doc, log, win) {
 	'use strict';
-	var logGroup = 'ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory';
+	var logGroup = 'ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory',
+		imaViewMode = {};
```

Comment:
I don't really like we're keeping imaViewMode in global scope, I need a moment to think whether it's no other way to do that

Reply:
see below